File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Ignore_TestResult.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

import com.google.gson.annotations.SerializedName;

data class Ignore_TestResult(
  val policy: PolicyEnum, // Oneof: ignore, use
) {

  enum class PolicyEnum {
    @SerializedName("ignore") Ignore,
    @SerializedName("use") Use,
  }
}

File: agent/bindings/csharp/IgnoreTestResult.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class IgnoreTestResult
  {
    [JsonProperty(PropertyName = "policy")]
    public PolicyEnum Policy { get; set; } // Oneof: ignore, use

    public enum PolicyEnum
    {
      [EnumMember(Value = "ignore")]
      Ignore,
      [EnumMember(Value = "use")]
      Use,
    }
  }
}
